### DISCUSSION TITLE
Shuffle command

### DISCUSSION THREAD

**Discussion by ADUS921** [hmes98318/Music-Disc#6] [Ideas]
Hello, have you thought about adding !shuffle command for shuffling queue?

**Comment by hmes98318** [comment]
> Hello, have you thought about adding !shuffle command for shuffling queue?

Sure, I will add it in the next version.

**Comment by hmes98318** [comment]
> Hello, have you thought about adding !shuffle command for shuffling queue?

Updated new version.
